### DISCUSSION TITLE
Pré-commande livre

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#322] [General]
Bonjour tout le monde,

Une pré-commande de la parution du livre est possible à un tarif préférentiel (10,90 au lieu de 12,90 prix public prévu par l'éditeur) pour quelques centaines de livres.  Est-ce que cela intéresse des personnes?

Si oui, n'hésitez pas à vous manifester rapidement avant diffusion plus large.

**Comment by davidgroult** [comment]
Hello ! 

Merci pour l'info ! 
Oui, bien sûr, intéressé de mon côté ! :)

**Reply by davidgroult** [reply]
2 exemplaires également :)

**Comment by arnolem** [comment]
Vas-y pour 2 exemplaires 👍

**Comment by rlemaire** [comment]
Hello!

Oui je suis bien sur intéressé. Je reviens prochainement avec un nombre
d'exemplaire après avoir échangé avec quelques collègues :-)

Bonne journée,
--
*Raphaël Lemaire*
***@***.***
*06 70 59 26 34*

**Comment by ACTLEM** [comment]
Merci pour vos retours. Une communication a finalement été faite sur la mailing list avec les informations pour profiter de cette pré-commande.
